From 2ec7f5d611ebc2fa6bea65d88ce7c4165a7b5eac Mon Sep 17 00:00:00 2001
From: Sergei Shkonda <sergeish@marvell.com>
Date: Tue, 8 Aug 2017 13:25:09 +0300
Subject: [PATCH 1217/1345] dts: a7020-amc: increase PCIe map window address
 range

commit  230ef7ca05c5f382e9e62c7addc6f6369a711b27 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Increase amount of addresses to be used for PCIe resource mappings
from 64M to 96M. This change required for CPSS project
(PCIe BARs sizes are 1M+64M+8M)

Change-Id: I113a3888e35456441219fb90a4b40afa7e88130d
Signed-off-by: Sergei Shkonda <sergeish@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/42714
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7020-amc.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7020-amc.dts b/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
index 8604534..1776545 100644
--- a/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7020-amc.dts
@@ -187,7 +187,7 @@
 		pcie@0x600000 {
 			/* overwrite the PCIe memory address since it request for 64MB */
 			ranges = <0x81000000 0 0xfa000000 0  0xfa000000 0 0x10000 /* downstream I/O */
-				  0x82000000 0 0xf6000000 0  0xf6000000 0 0x4000000>; /* non-prefetchable memory */
+				  0x82000000 0 0xf6000000 0  0xf6000000 0 0x6000000>; /* non-prefetchable memory */
 			num-lanes = <4>;
 			status = "okay";
 		};
-- 
1.7.9.5

